includes/template-library/manager.php

```diff
@@ -227,9 +227,10 @@ public function get_library_data( array $args ) {
 		Plugin::$instance->documents->get_document_types();
 
 		$filter_sources = ! empty( $args['filter_sources'] ) ? $args['filter_sources'] : [];
+		$force_update = ! empty( $args['sync'] );
```

Comment:
Does sync always have a value or it might also be `?param1=value&param2=value&sync&param3=value` ?
If so, should switch to `isset()`

Reply:
Line 220, Kobi used it like that, I did the same :)

Reply:
Ahm, ok